src/ucs/config/parser.c

```diff
@@ -1420,6 +1421,44 @@ static ucs_status_t ucs_config_parser_get_sub_prefix(const char *env_prefix,
     return UCS_OK;
 }
 
+void ucs_config_parse_config_files()
+{
+    char file_path[MAXPATHLEN];
+    const char *dir_path;
+
+    /* System-wide configuration file */
+    ucs_snprintf_safe(file_path, MAXPATHLEN, "%s/%s", UCX_CONFIG_DIR,
+                      UCX_CONFIG_FILE_NAME);
+    ucs_config_parse_config_file(file_path, 1);
+
+    /* Library dir */
+    dir_path = ucs_sys_get_lib_path();
+    if (dir_path != NULL) {
+        ucs_snprintf_safe(file_path, MAXPATHLEN, "%s/../etc/%s", dir_path,
+                          UCX_CONFIG_FILE_NAME);
+        ucs_config_parse_config_file(file_path, 1);
```

Comment:
i think ucs_snprintf_safe can be moved inside ucs_config_parse_config_file
in this particular case, can call 
`ucs_config_parse_config_file(dir_path, "../etc/" UCX_CONFIG_FILE_NAME, 1);`
and for last case:
`ucs_config_parse_config_file(".", UCX_CONFIG_FILE_NAME, 1);`

Reply:
Fixed